CMakeLists.txt

```diff
@@ -27,6 +27,25 @@ list( APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake )
 project( nest CXX C )
 set( NEST_USER_EMAIL "users@nest-simulator.org" )
 
+include( ColorMessages )
+
+# check if the given CMAKE_INSTALL_PREFIX is not empty
```

Comment:
This comment doesn't actually match with the if condition. Would it make more sense to check for empty string outside the `if( NOT IS_ABSOLUTE ${CMAKE_INSTALL_PREFIX})` since this condition is not a pre-requisite for checking empty string?